specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-09-01/afdx.json

```diff
@@ -9442,7 +9442,7 @@
           "minimum": 16,
           "exclusiveMinimum": false
         },
-        "state": {
+        "enabledState": {
```

Comment:
Why was this renamed from state to enabledState?

Reply:
because it was confusing when I was writing the RP code for the models. In our current existing DataModels, "State" refers to resource state, and not whether it's enabled or not. Having the word "enabled" here can explicitly tell us what this state is referring to.